content/miq_dialogs/miq_provision_azure_dialogs_template.yaml

```diff
@@ -357,9 +357,23 @@
           :required: true
           :display: :edit
           :data_type: :string
+          :required_method: :validate_blacklist
+          :blacklist: [
+            'administrator', 'admin', 'user', 'user1', 'test', 'user2', 'test1', 'user3',
+            'admin1', '1', '123', 'a', 'actuser', 'adm', 'admin2', 'aspnet', 'backup',
+            'console', 'david', 'guest', 'john', 'owner', 'root', 'server', 'sql',
+            'support', 'support_388945a0', 'sys', 'test2', 'test3', 'user4', 'user5'
```

Comment:
I see this list is taken directly from the MS doc linked in the description but it would be easier to read if similar items were grouped together?  Like all the `admin` based ones and all the `user#` ones.

Just curious if you have any idea why they blacklist `support_388945a0`, `david` and `john`?  They seem odd compared to something obvious like `admin` and `user1`.